src/Form/FormAnalyzer.js

```diff
@@ -383,6 +382,17 @@ class FormAnalyzer {
         this._isCCForm = Boolean(textMatches && deDupedMatches.size > 1);
         return this._isCCForm;
     }
+
+    /**
+     * @param {string} text
+     * @returns {boolean}
+     */
+    shouldFlipScoreForButtonText(text) {
```

Comment:
I wanted to re-use this also for external link part, but that's a little bit different in terms of score strength. keeping it